[network] Tests failing

Build on Windows 10:
```
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running org.openhab.binding.network.internal.dhcp.DHCPTest
[Thread-1] INFO org.openhab.binding.network.internal.dhcp.DHCPPacketListenerServer - DHCP request packet listener online
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.463 s - in org.openhab.binding.network.internal.dhcp.DHCPTest
[INFO] Running org.openhab.binding.network.internal.discovery.DiscoveryTest
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.727 s - in org.openhab.binding.network.internal.discovery.DiscoveryTest
[INFO] Running org.openhab.binding.network.internal.handler.NetworkHandlerTest
[Thread-2] INFO org.openhab.binding.network.internal.dhcp.DHCPPacketListenerServer - DHCP request packet listener online
[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.285 s - in org.openhab.binding.network.internal.handler.NetworkHandlerTest
[INFO] Running org.openhab.binding.network.internal.PresenceDetectionTest
[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.247 s - in org.openhab.binding.network.internal.PresenceDetectionTest
[INFO] Running org.openhab.binding.network.internal.PresenceDetectionValuesTest
[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.network.internal.PresenceDetectionValuesTest
[INFO] Running org.openhab.binding.network.internal.toberemoved.cache.ExpiringCacheAsyncTest
[ERROR] Tests run: 5, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.147 s <<< FAILURE! - in org.openhab.binding.network.internal.toberemoved.cache.ExpiringCacheAsyncTest
[ERROR] testExpiring(org.openhab.binding.network.internal.toberemoved.cache.ExpiringCacheAsyncTest)  Time elapsed: 0.147 s  <<< FAILURE!
org.mockito.exceptions.verification.WantedButNotInvoked:

Wanted but not invoked:
expiringCacheUpdate.requestCacheUpdate();
-> at org.openhab.binding.network.internal.toberemoved.cache.ExpiringCacheAsyncTest.testExpiring(ExpiringCacheAsyncTest.java:89)
Actually, there were zero interactions with this mock.

        at org.openhab.binding.network.internal.toberemoved.cache.ExpiringCacheAsyncTest.testExpiring(ExpiringCacheAsyncTest.java:89)

[INFO] Running org.openhab.binding.network.internal.utils.LatencyParserTest
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 s - in org.openhab.binding.network.internal.utils.LatencyParserTest
[INFO]
[INFO] Results:
[INFO]
[ERROR] Failures:
[ERROR]   ExpiringCacheAsyncTest.testExpiring:89
Wanted but not invoked:
expiringCacheUpdate.requestCacheUpdate();
-> at org.openhab.binding.network.internal.toberemoved.cache.ExpiringCacheAsyncTest.testExpiring(ExpiringCacheAsyncTest.java:89)
Actually, there were zero interactions with this mock.

[INFO]
[ERROR] Tests run: 22, Failures: 1, Errors: 0, Skipped: 0
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 10.041 s
[INFO] Finished at: 2020-06-22T18:18:16+02:00
[INFO] Final Memory: 36M/657M
[INFO] ------------------------------------------------------------------------
```

--- a/bundles/org.openhab.binding.network/src/test/java/org/openhab/binding/network/internal/toberemoved/cache/ExpiringCacheAsyncTest.java
+++ b/bundles/org.openhab.binding.network/src/test/java/org/openhab/binding/network/internal/toberemoved/cache/ExpiringCacheAsyncTest.java
@@ -78,7 +78,7 @@ public class ExpiringCacheAsyncTest {
         verify(u, times(0)).requestCacheUpdate();
         // Wait
         try {
-            Thread.sleep(100);
+            Thread.sleep(101);
         } catch (InterruptedException ignored) {
             return;
         }

